petastorm/unischema.py

```diff
@@ -234,9 +238,15 @@ def from_arrow_schema(cls, parquet_dataset):
         for column_name in arrow_schema.names:
             arrow_field = arrow_schema.field_by_name(column_name)
             field_type = arrow_field.type
-            codec, np_type = _numpy_and_codec_from_arrow_type(field_type)
-
-            unischema_fields.append(UnischemaField(column_name, np_type, (), codec, arrow_field.nullable))
+            try:
+                codec, np_type = _numpy_and_codec_from_arrow_type(field_type)
```

Comment:
Please reduce the scope of try/except to the line that raises the error to avoid wrong behavior if line 243 raises `ValueErorr`

Reply:
Ok